scripts/repochecker

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
```

Comment:
IMHO, the set -e is a bit useless here, as you add || true to most commands except the last two ones and maybe mkdir .
you could use something like the || complain $? "message"  from mkcloud on the few lines that need exit-code checking